package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "discord.js",
-  "version": "11.5.1",
+  "version": "11.5-dev",
```

```suggestion
  "version": "11.6.0-dev",
```
This needs to be `11.6.0-dev`